src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -48,7 +48,9 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
 
         CommandResult commandResult;
         Command command;
-        if (commandText.startsWith(TaskCommand.COMMAND_WORD) && Character.isWhitespace(commandText.charAt(4))) {
+        if ((commandText.startsWith(TaskCommand.COMMAND_WORD) && Character.isWhitespace(commandText.charAt(4)))
+            || (commandText.startsWith(TaskCommand.COMMAND_WORD_ALIAS)
+                && Character.isWhitespace(commandText.charAt(1)))) {
```

Comment:
Could we abstract this better instead of using hardcoded values to test for whitespace? At least use COMMAND_WORD.length() instead. But it might be cleaner to write a helper function that does it for any given string!